### DISCUSSION TITLE
July 17: Webinar QField Plugin Framework – Redefine Your Field Work!

### DISCUSSION THREAD

**Discussion by ZsanettMed** [opengisch/QField#5434] [Show and tell]
Dive into the capabilities of the QField Plugin Framework and learn how it can transform your fieldwork experience at our upcoming webinar on July 17, 3 PM CEST. 

[Register your seat now!](https://forms.clickup.com/2192114/f/22wqj-35495/8X7D0GB303ZDD9VL39)

**Comment by mbernasocchi** [comment]
The Webinar is fully booked, but we have opened a waiting list. You can register normally, and we'll invite you if a place opens up.

**Reply by mahmoodi-source** [reply]
Hi, 
Firstly, I would like to thank you for the very well-organized and informative session.
Secondly, we were supposed to receive a copy of the recorded video, which we have not yet received.
Thanks.
